client/components/overview/Overview.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+import Summary from './Summary.jsx';
+import StarDistribution from './StarDistribution.jsx';
+import LovedFor from './LovedFor.jsx';
+import exampleData from './../exampleData.js';
+import overview from './helperFunctions.jsx';
+
+class Overview extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      totalReviews : 1381,
+      neighborhood : 'SOMA',
+      overallRating : overview.overallRating(exampleData),
+      foodRating : overview.foodRating(exampleData),
+      serviceRating : overview.serviceRating(exampleData),
+      ambienceRating : overview.ambienceRating(exampleData),
+      valueRating : overview.valueRating(exampleData),
+      noiseLevel : overview.noiseLevel(overview.noise(exampleData)),
+      recommended : overview.recommended(exampleData)
+    }
+  }
+
+
+  render() {
+    axios.get('/reviews')
```

Comment:
I know that this is a stubbed out version of axios get request so it's not doing anything functional right now, but it may not good placement of it for the future. Feel free to message me and we can talk about why and other potential options.